src/harness/reference_models/iap/test_data/cbsd_2.json

```diff
@@ -0,0 +1,44 @@
+{
+        "_comment":"FSS - TT&C Blocking CBSD",
+	"id": "device_2_id",
+	"registrationRequest": {
+					"fccId": "test_fcc_id_c",
+		            "cbsdCategory": "B",
```

Comment:
Change the CBSD category to A or set the indoor deployment flag to false

Reply:
Updated